logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -142,7 +149,8 @@ sealed class ConversationDetails(open val conversation: Conversation) {
         val unreadMessagesCount: Long = 0L,
         val unreadMentionsCount: Long = 0L,
         val lastUnreadMessage: Message?,
-        val isSelfUserMember: Boolean = true
+        val isSelfUserMember: Boolean = true,
+        val isSelfCreated: Boolean = false
```

Comment:
what is the difference between isCreated and isSelfCreated ?

Reply:
You mean `isSelfUserMember`?
I guess this one means that self user is a member of the conversation when true. False would mean that self user left or was removed from the conversation.

And `isSelfCreated` could mean that self user created the conversation.